node_modules/oae-content/lib/backends/local.js

```diff
@@ -14,7 +14,7 @@
  */
 
 var fs = require('fs');
-var mkdirp = require('mkdirp');
+const makeDir = require('make-dir');
```

Comment:
We could probably make the other dependencies `const` as well while we're at it.